assets/bundles/bundle.properties

```diff
@@ -253,40 +256,40 @@ heimdall.force-field-restore-time.description=How long shield prepares after bro
 heimdall.power-use.name=Power Use
 heimdall.power-use.description=The use of power per second.
 # Heimdall Killing
-heimdall.msg.killing-0=Salvation
-heimdall.msg.killing-1=Well done
-heimdall.msg.killing-2=They will remember us
-heimdall.msg.killing-3=Nice killing
+heimdall.msg.killing-0=Salvation.
+heimdall.msg.killing-1=Well done.
+heimdall.msg.killing-2=They will remember us.
+heimdall.msg.killing-3=Nice killing!
 # Heimdall Control
-heimdall.msg.control-0=Next one
-heimdall.msg.control-1=Now it's allegiance to us
-heimdall.msg.control-2=Another one
-heimdall.msg.control-3=One after another
-heimdall.msg.control-4=Obey us
-heimdall.msg.control-5=Come here
-heimdall.msg.control-6=Welcome
-heimdall.msg.control-7=New body
-heimdall.msg.control-8=Collective
+heimdall.msg.control-0=Next one.
+heimdall.msg.control-1=Now it's allegiance to us…
+heimdall.msg.control-2=Another one.
+heimdall.msg.control-3=One after another.
+heimdall.msg.control-4=Obey us!
+heimdall.msg.control-5=Come here…
+heimdall.msg.control-6=Welcome!
+heimdall.msg.control-7=New body.
+heimdall.msg.control-8=Collective.
 heimdall.msg.control-in-mod-0=Who is this?
-heimdall.msg.control-in-mod-1=Stranger
-heimdall.msg.control-in-mod-2=Never seen
+heimdall.msg.control-in-mod-1=Stranger.
+heimdall.msg.control-in-mod-2=Never seen this.
 # Heimdall Detect Enemy
-heimdall.msg.detect-0=Something is coming
+heimdall.msg.detect-0=Something is coming!
 heimdall.msg.detect-1=Who is coming?
-heimdall.msg.detect-2=Fire
-heimdall.msg.detect-controlled-0=We will win
-heimdall.msg.detect-controlled-1=What are we waiting for
-heimdall.msg.detect-controlled-2=Shoot, now
-heimdall.msg.detect-controlled-3=We together
+heimdall.msg.detect-2=Fire!
+heimdall.msg.detect-controlled-0=We will win!
+heimdall.msg.detect-controlled-1=What are we waiting for?
+heimdall.msg.detect-controlled-2=Shoot, now!
+heimdall.msg.detect-controlled-3=We, together.
 # Heimdall Controlled
-heimdall.msg.controlled-0=How many me
-heimdall.msg.controlled-1=......
-heimdall.msg.controlled-2=Who am I
-heimdall.msg.controlled-3=What am I
-heimdall.msg.controlled-4=Where am I
-heimdall.msg.controlled-5=Only me...
-heimdall.msg.controlled-6=I can feel myself
-heimdall.msg.controlled-7=[Thinking...]
+heimdall.msg.controlled-0=How many are we?
+heimdall.msg.controlled-1=…
```

Comment:
The length of string will affect the duration of animation.

Reply:
i know